docs/api.md

```diff
@@ -56,7 +104,7 @@
 
 ```{eval-rst}
 .. automodule:: pytket.extensions.qiskit.backends.crosstalk_model
-    :members: CrosstalkParams
+    :members: CrosstalkParams, NoiseGate, NoisyCircuitBuilder, FractionalUnitary, get_gate_times_from_backendinfo
```

Comment:
@yao-cqc Do you know which of these classes are intended to be public?

Reply:
Will do. Btw, `get_gate_times_from_backendinfo` is not used anywhere, can I remove it entirely?